packages/@ourworldindata/grapher/src/tooltip/TooltipProps.ts

```diff
@@ -33,13 +38,15 @@ export interface TooltipValueProps {
     value?: number | string
     color?: string
     notice?: number | string // actual year data was drawn from (when ≠ target year)
+    superscript?: boolean // show significance-s superscript if applicable
```

Comment:
I think I would misunderstand what superscript could mean here - better give it a longer name involving significance rounding in some form?

Reply:
Renamed it to `showSignificanceSuperScript` – hope that is a bit clearer.